spec/lib/internal_metric/global_spec.rb

```diff
@@ -112,5 +112,14 @@ module DiscoursePrometheus::InternalMetric
         expect(metric.postgres_replica_available).to eq(0)
       end
     end
+
+    it "can collect pg_seq metric" do
+      3.times { Fabricate(:user) }
```

Comment:
Just curious, instead of fabricating.. I wonder if we can just hardcode the sequence with [ALTER SEQUENCE](https://www.postgresql.org/docs/current/sql-altersequence.html). We'll have to reset it to the original value after the test though.

Reply:
Yeah we probably could, but I'm not 100% sure of the possible side effects. Using a standard `Fabricate` seems much less risky and also makes it a more realistic test.

Reply:
The problem is that rolling back does not actually reset the sequence so if other tests which are fabricating user records before this test is ran, our assertion will not be accurate. But you're right that messing with sequence is probably not a good idea. Maybe we can look up the current sequence value first, fabricate and then assert against the expected sequence value?